server/server.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path')
+const express = require('express')
+const app = express()
+// const publicPath = path.join(__dirname, '..', 'public');
+const publicPath = path.join(__dirname, '..', 'build')
+const port = process.env.PORT || 3000
+app.use(express.static(publicPath))
+
+app.get('*', (req, res) => {
+  res.sendFile(path.join(publicPath, 'index.html'))
+})
+
+app.listen(port, () => {
+  console.log('Server is up!')
+})
```

Comment:
**NOTE:**
Creating your react app with `npx create-react-app` doesn't require you to set up `node.js` server.

For some reason, you included the `node.js` server settings.

I'm curious, am I missing something?